tests/test_api/test_export.py

```diff
@@ -1,41 +1,38 @@
-from django.contrib.auth.models import User
-from django.test.client import Client
 from django.urls import reverse
 
+from rest_framework import status
+from rest_framework.test import APIClient
```

Comment:
```suggestion
from rest_framework import status, test
```

And then test.APIClient